### PR TITLE
[ADD][brd_project_categ] business requirement deliverable project categ

### PR DESCRIPTION
**Pull request by martintorresvm**
To continue work done on #59

### CODE REVIEW

**Comment by martintorresvm** [comment]
@elicoidal I would suggest close #59 as it will continue or this PR.

**Comment by martintorresvm** [comment]
All test has passed. Please review @elicoidal @seb-elico

File: business_requirement_deliverable_project_categ/README.rst [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback.
+
+Credits
+=======
+
+Contributors
+------------
+
+* Eric Caudal<eric.caudal@elico-corp.com>
```
Comment (seb-elico) [inline business_requirement_deliverable_project_categ/README.rst]: @victormartinelicocorp there's no space between name and email, Travis doesn't check this?

**Review by seb-elico — CHANGES_REQUESTED** [review]

File: business_requirement_deliverable_project_categ/README.rst (hunk 1 above)
Comment (martintorresvm) [reply business_requirement_deliverable_project_categ/README.rst]: Me neither, if Travis could check format of rst files never face before. But indeed I need to fix it

@gurneyalex Could you help us with this question?

**Review by elicoidal — APPROVED** [review]

**Comment by elicoidal** [comment]
@seb-elico please recheck

**Comment by martintorresvm** [comment]
@elicoidal @seb-elico we actually are using this module included I just fix Frank's code so could be possible merged and switch to OCA. 

Please let me know if we could move forward.

**Comment by elicoidal** [comment]
@victormartinelicocorp OK for me.
The module will be merged soon in OCA anyway

**Review by seb-elico — APPROVED** [review]

**Comment by seb-elico** [comment]
@elicoidal @victormartinelicocorp All good on my side. Runbot cover decreased of 0.3%, I don't know if it's a blocking point

**Comment by martintorresvm** [comment]
> @elicoidal @victormartinelicocorp All good on my side. Runbot cover decreased of 0.3%, I don't know if it's a blocking point

@seb-elico  `Coveralls` is a little deficient on giving good analysis but if you check the message on `covecov/project` then you'll see that for this PR is normal.

`Absolute coverage decreased by -0.25% but relative coverage increased by +6.19%`

**Comment by elicoidal** [comment]
@pedrobaeza @dreispt @seb-elico Let's move forward?

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**